sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOrcTimezoneUtils.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids
+
+import java.util.TimeZone
+
+object GpuOrcTimezoneUtils {
+
+  def isGpuOrcSupportedTimeZone(jvmTz: TimeZone): Boolean = {
+    jvmTz == TimeZone.getTimeZone("UTC") || jvmTz == TimeZone.getTimeZone("Asia/Shanghai")
```

Comment:
Why only these two? I don't see anywhere that we could not support all time zones.